tests/b2share_unit_tests/upgrade/helpers.py

```diff
@@ -95,6 +95,9 @@ def check_records_migration():
             'type': RecordUUIDProvider.parent_pid_type,
             'value': parent_pid.pid_value,
         })
+        # The OAI-PMH identifier has been modified by the migration
+        if db_record.get('_oai'):
+            exp_record['metadata']['_oai']['id'] = db_record['_oai']['id']
```

Comment:
The OAI ID is deterministic. It would be better to build the expected OAI ID instead of using the one from the DB, otherwise we don't know if we added a bug. We could use a minter on an empty dict and extract the expected PID.

Reply:
You're right.